magmi/inc/magmi_engine.php

```diff
@@ -513,6 +513,17 @@ public function connectToMagento()
                 $user = $default_setup->username;
                 $pass = $default_setup->password;
                 $port = $default_setup->port;
+            } else if ($conn == 'envphp') { //berto - scraping env.php
```

Comment:
Please remove this comment too, and use tripple ===

Reply:
done